giskard_cicd/pipeline/scan_config.yaml

```diff
@@ -0,0 +1,13 @@
+detectors:
+  - ethical_bias
+  - text_perturbation
+  - robustness
+  - performance
+  - underconfidence
+  - overconfidence
+  - spurious_correlation
+
```

Comment:
💡💭: We skip data leakage detector because it requires too many inference api calls.

Reply:
TBD